src/components/DoughnutChart/DoughnutChart.scss

```diff
@@ -0,0 +1,30 @@
+@import "vanilla-framework";
+
+.doughnut-chart {
+  width: 6.5rem;
+
+  .doughnut-chart__tooltip {
+    display: block;
+  }
+
+  .doughnut-chart__tooltip > :only-child {
```

Comment:
I can't say that I know exactly why this occurs, but when this section is removed, the chart fails to render upon hover. It simply just disappears :skull: , so this part prevents that from happening.
![image](https://github.com/user-attachments/assets/1c261dc9-7a29-41c7-9c20-aea28b2196e1)

As for the Tooltip display:block, this can be removed, though I'm not sure if this is what you were referring to as it is not an inline-block statement.